docs/user-guide/rules.md

```diff
@@ -0,0 +1,467 @@
+- [`font-family-no-duplicate-names`](../../lib/rules/font-family-no-duplicate-names/README.md): Disallow duplicate names within font families.
+- [`keyframe-block-no-duplicate-selectors`](../../lib/rules/keyframe-block-no-duplicate-selectors/README.md): Disallow duplicate selectors within keyframe blocks.
+- [`no-duplicate-at-import-rules`](../../lib/rules/no-duplicate-at-import-rules/README.md): Disallow duplicate `@import` rules.
+- [`no-duplicate-selectors`](../../lib/rules/no-duplicate-selectors/README.md): Disallow duplicate selectors.
+
+### Descending
+
+Disallow descending things with these `*-no-descending` rules.
+
+- [`no-descending-specificity`](../../lib/rules/no-descending-specificity/README.md): Disallow selectors of lower specificity from coming after overriding selectors of higher specificity.
+
+### Empty
+
+Disallow empty things with these `*-no-empty` rules.
+
+- [`block-no-empty`](../../lib/rules/block-no-empty/README.md): Disallow empty blocks.
+- [`comment-no-empty`](../../lib/rules/comment-no-empty/README.md): Disallow empty comments.
+- [`no-empty-source`](../../lib/rules/no-empty-source/README.md): Disallow empty sources.
+
+### Irregular
+
+Disallow irregular things with these `*-no-irregular` rules.
+
+- [`no-irregular-whitespace`](../../lib/rules/no-irregular-whitespace/README.md): Disallow irregular whitespace.
+
+### Invalid
+
+Disallow invalid syntax with these (sometimes implicit) `no-invalid` rules.
+
+- [`color-no-invalid-hex`](../../lib/rules/color-no-invalid-hex/README.md): Disallow invalid hex colors.
+- [`function-calc-no-unspaced-operator`](../../lib/rules/function-calc-no-unspaced-operator/README.md): Disallow invalid unspaced operator within `calc` functions (Autofixable).
+- [`keyframe-declaration-no-important`](../../lib/rules/keyframe-declaration-no-important/README.md): Disallow invalid `!important` within keyframe declarations.
+- [`named-grid-areas-no-invalid`](../../lib/rules/named-grid-areas-no-invalid/README.md): Disallow invalid named grid areas.
+- [`no-invalid-double-slash-comments`](../../lib/rules/no-invalid-double-slash-comments/README.md): Disallow invalid double-slash comments.
+- [`no-invalid-position-at-import-rule`](../../lib/rules/no-invalid-position-at-import-rule/README.md): Disallow invalid position `@import` rules.
+- [`string-no-newline`](../../lib/rules/string-no-newline/README.md): Disallow invalid newlines within strings.
+
+### Missing
+
+Disallow missing things with these `*-no-missing` rules.
+
+- [`custom-property-no-missing-var-function`](../../lib/rules/custom-property-no-missing-var-function/README.md): Disallow missing `var` function for custom properties.
+- [`font-family-no-missing-generic-family-keyword`](../../lib/rules/font-family-no-missing-generic-family-keyword/README.md): Disallow missing generic family keyword within font families.
+
+### Nonstandard
+
+Disallow nonstandard things with these `*-no-nonstandard` rules.
```

Comment:
[suggest] The word "non-standard" seems a little more readable, although the rule name is unchangeable.

```suggestion
### Non-standard

Disallow non-standard things with these `*-no-nonstandard` rules.
```